css-table/table.css

```diff
@@ -0,0 +1,72 @@
+body {
+	font-family: sans-serif;
+	background-color: #f0f8ff;
+}
+tbody {
+	display: block; 
```

Comment:
You can remove it. It is not required

Reply:
Hi @Neha 
    If I remove the tbody css property , then fixed header will not work , Now, do i want to remove this css property and try it in different way to fix the header ?

Thanks,
keerthanaIT :-)

Reply:
Done

Reply:
> Hi @Neha
> Done with All the changes , Please review it once .
> 
> Thanks,
> KeerthanaIT :-)

Thank you @keerthanaIT , it looks good. I hope you learned something new from this task :)